src/components/SectionHeading/index.tsx

```diff
@@ -8,9 +8,16 @@ export interface SectionHeadingProps {
   heading: string;
   rightContent?: ReactNode;
   subHeading?: string;
+  isDisplayLine?: boolean;
```

Comment:
Do you think if this would be a better name?
```suggestion
  renderLine?: boolean;
```

Or maybe
```suggestion
  showLine?: boolean;
```